### PR TITLE
[#151022906] Send generic JSON payloads from web endpoint to a queue owned by Data team

### PR DESCRIPTION
**Pull request by sergii-tkachenko**
#### What's this PR do?
- Implements web endpoint `/api/v1/events/quasar-relay`
- Saves all JSON payloads received by the endpoint to `quasar-customer-io-email-activity` queue (a subject to change changed as described in #99)

#### How should this be manually tested?
- `yarn install`
- `docker-compose up`
- `yarn all-tests`
- `yarn web`
- Send freeform JSON to `/api/v1/events/quasar-relay`
- Make sure you can see the payload in http://localhost:15672/#/queues/blink/customer-io-update-customer -> Get Messages

#### Any background context you want to provide?
For now, `quasar-customer-io-email-activity` queue is used for a generic integration with Data consumer. It will be renamed to `data-prime` queue or `quasar-data-prime` soon, see #99.

#### What are the relevant tickets?
Pivotal [#151022906](https://www.pivotaltracker.com/story/show/151022906)
A part of #127

### CODE REVIEW

File: test/helpers/MessageFactoryHelper.js [hunk 1]
```diff
@@ -244,6 +244,28 @@ class MessageFactoryHelper {
       meta: {},
     });
   }
+
+  static getRandomDataSample(nested = false) {
+    const data = {};
+
+    // Add random words.
+    for (let i = 0; i < 8; i++) {
```
Comment (rapala61) [inline test/helpers/MessageFactoryHelper.js]: This made me miss Ruby ranges. I found a nifty alternative in JS `[...Array(5).keys()]`. Could help not having to add exception for eslint `id-length`.

Comment (sergii-tkachenko) [reply test/helpers/MessageFactoryHelper.js]: I kinda like classic fors too :)

Comment (sergii-tkachenko) [reply test/helpers/MessageFactoryHelper.js]: And the exception for `i` [is actually there](https://github.com/DoSomething/eslint-config/blob/master/index.js#L19), but [I've overridden](https://github.com/DoSomething/eslint-config/blob/master/nodejs/ava.js#L40) it in ava-specific settings inadvertently.

**Comment by sergii-tkachenko** [comment]
Looks like we're publishing 2 times at different places to the same queue:
- POST /api/v1/webhooks/customerio-email-activity should publish message to customer-io queue
- POST /api/v1/events/quasar-relay should save message as is to quasar queue

Because of this race condition tests fail from time to time.

**Pull request merged**